### DISCUSSION TITLE
Usage of SignTypedDatav4

### DISCUSSION THREAD

**Discussion by pedrohba1** [wevm/wagmi#989] [Q&A]
Currently, the hook `useSignTypedData` doesn't seem to support the latest version of the sign typed data implementation: https://docs.metamask.io/guide/signing-data.html#sign-typed-data-v3

Is there a workaround to use it with wagmi? How can we implement it?

**Comment by tmm** [comment]
It should support the latest version of the spec. What makes you think it doesn't?
